analyzers/ResonanceBuilder.py

```diff
@@ -49,5 +49,8 @@ def process(self, event):
             resonances.append( Resonance(leg1, leg2, self.cfg_ana.pdgid) )
         # sorting according to distance to nominal mass
         nominal_mass = mass[self.cfg_ana.pdgid]
-        resonances.sort(key=lambda x: abs(x.m()-nominal_mass))
+        if self.cfg_ana.pdgid==32:
```

Comment:
This should not be necessary, there is no reason to treat a given type of resonance differently

Reply:
I ll check why we have done this